app/assets/stylesheets/layouts/_footer.scss

```diff
@@ -4,4 +4,13 @@
   height: $baseline-unit*20;
   border-top: $default-border-thin;
   background-color: $background-color--light;
+
+  p {
+    @include body(14,18);
+  }
 }
+
+.l-footer__inner {
+  @include column(12);
+  padding-top: $baseline-unit*4;
+}
```

Comment:
I find YML files more readable if you can use whitespace to divide sections. @benlovell thoughts?